src/Utils.ts

```diff
@@ -123,6 +123,10 @@ export default class Utils {
         });
     }
 
+    static getHomeDirectory(): string {
```

Comment:
I can see you use the `get` syntax in the other place. Maybe we should do it here as well?